src/main/java/frc/team2412/robot/commands/intake/AllInCommand.java

```diff
@@ -30,6 +33,7 @@ public void end(boolean interrupted) {
 		intakeSubsystem.intakeReject();
 		intakeSubsystem.indexStop();
 		intakeSubsystem.feederStop();
+		controls.vibrateDriveController(1.0);
```

Comment:
Since the command can be interrupted for reasons other than `isFinished()` being signaled (could be aborted if scheduled in parallel with a wait command, a different button is pressed that starts a different command using the intake subsystem, etc.), it'd be best to check and only rumble the controller if `intakeSubsystem.feederSensorHasNote()` is true.

Reply:
Or, more simply, only vibrate the controller if the `interrupted` parameter is `false` (meaning the command is ending because it finished and wasn't interrupted).